notifier.rst

```diff
@@ -125,21 +125,26 @@ The chat channel is used to send chat messages to users by using
 :class:`Symfony\\Component\\Notifier\\Chatter` classes. Symfony provides
 integration with these chat services:
 
-==========  ===============================  ============================================
+==========  ===============================  ================================================
 Service     Package                          DSN
-==========  ===============================  ============================================
+==========  ===============================  ================================================
 Slack       ``symfony/slack-notifier``       ``slack://default/ID``
 Telegram    ``symfony/telegram-notifier``    ``telegram://TOKEN@default?channel=CHAT_ID``
 Mattermost  ``symfony/mattermost-notifier``  ``mattermost://TOKEN@ENDPOINT?channel=CHANNEL``
 RocketChat  ``symfony/rocketchat-notifier``  ``rocketchat://TOKEN@ENDPOINT?channel=CHANNEL``
-==========  ===============================  ============================================
+Zulip       ``symfony/zulip-notifier``       ``zulip://EMAIL:APIKEY@ENDPOINT?channel=STREAM``
```

Comment:
I would change `...?channel=STREAM` to `...?channel=CHANNEL` to be consistent with the other transport DSNs.

Reply:
@jschaedl done, thanks for the suggestion!